### PR TITLE
Adds website packaging support to release assets into optimizely/ via grunt release

### PR DESCRIPTION
**Pull request by brianlucas**
This allows you to bundle up your website release and push it out to github.  You can check the releases here:

https://github.com/optimizely/marketing-website/releases

To run this process, type 'grunt release' and enter your username and API password.  You may need to create a github API password here:
https://github.com/settings/applications

### CODE REVIEW

**Comment by brianlucas** [comment]
cc @kylerush

**Comment by kylerush** [comment]
@brianlucas i'm not sure it's practical for us to run this for every merge we do, is that the expectation? i've used tags before which work great for bigger applications with fewer deploys, but we deploy often and we deploy pull requests from marketers that don't work in the command line and we don't test their stuff in the command line. so this would add quite a bit of bulk to our process. do i have that wrong?

**Comment by brianlucas** [comment]
No, it’s completely optional and can be run anytime you want to push updates into the optimizely app (for testing, etc).

It doesn’t need to be run with every merge or release.

**Comment by kylerush** [comment]
@brianlucas gotcha, thanks

File: package.json [hunk 1]
```diff
@@ -47,9 +47,13 @@
   },
   "devDependencies": {
     "body-parser": "1.8.3",
+    "grunt-contrib-compress": "^0.13.0",
```
Comment (dtothefp) [inline package.json]: @brianlucas we like to constrain all our dependency versions, can you remove the carets?

File: Gruntfile.js [hunk 2]
```diff
@@ -133,5 +135,23 @@ module.exports = function(grunt) {
   grunt.registerTask('default', [
     'build'
   ]);
```
Comment (dtothefp) [inline Gruntfile.js]: @brianlucas I've never used `grunt.loadNpmTasks`, what does it do?

**Comment by dtothefp** [comment]
@brianlucas LGTM as soon as you constrain the package versions in package.json :neckbeard:

**Pull request merged**